### ISSUE TITLE
Bug report: msa::msaConvert to AAbin format results in incorrect FASTA file

### ISSUE THREAD

**Issue by richelbilderbeek** [UBod/msa#7]
Dear prof Bodenhofer, hello @UBod,

Here I submit a reproducible example of a bug report that proves that a call to `msa::msaConvert` to convert an `msa` alignment to `ape::AAbin` format results in something that `ape` cannot store properly. After that I post my `sessionInfo`.

The example is starts from the example code in the `msa::msaConvert documentation` to create an `ape::AAbin`, then simply saves it and shows the (unexpected) file content. 

I discovered this bug due to [a similar bug when saving to seqinr format](https://github.com/UBod/msa/issues/6).

Of course, I may have misunderstood something, even though the documentation of `msa::msaConvert` is quite clear. Additionally, I would be help to help verify if this is fixed :+1:

Good luck, Richel Bilderbeek

```r
# Use example code from msa::msaConvert
filepath <- system.file("examples", "exampleAA.fasta", package = "msa")
mySeqs <- readAAStringSet(filepath)
myAlignment <- msa(mySeqs)
msa_as_ape_aabin <- msa::msaConvert(myAlignment, "ape::AAbin")

# Save it to file
fasta_filename <- tempfile()
ape::write.FASTA(x = msa_as_ape_aabin, file = fasta_filename) # Gives warnings

# Show the file content. Below I show the output, notice there
# are no sequences saved
readLines(fasta_filename)
#  [1] ">PH4H_Rattus_norvegicus"         ""
#  [3] ">PH4H_Mus_musculus"              ""
#  [5] ">PH4H_Homo_sapiens"              ""
#  [7] ">PH4H_Bos_taurus"                ""
#  [9] ">PH4H_Chromobacterium_violaceum" ""
# [11] ">PH4H_Ralstonia_solanacearum"    ""
# [13] ">PH4H_Caulobacter_crescentus"    ""
# [15] ">PH4H_Pseudomonas_aeruginosa"    ""
# [17] ">PH4H_Rhizobium_loti"            ""
# Warning messages:
# 1: In readLines(fasta_filename) :
#   line 2 appears to contain an embedded nul
# 2: In readLines(fasta_filename) :
#   line 4 appears to contain an embedded nul
# 3: In readLines(fasta_filename) :
#   line 6 appears to contain an embedded nul
# 4: In readLines(fasta_filename) :
#   line 8 appears to contain an embedded nul
# 5: In readLines(fasta_filename) :
#   line 10 appears to contain an embedded nul
# 6: In readLines(fasta_filename) :
#   line 12 appears to contain an embedded nul
# 7: In readLines(fasta_filename) :
#   line 14 appears to contain an embedded nul
# 8: In readLines(fasta_filename) :
#   line 16 appears to contain an embedded nul
# 9: In readLines(fasta_filename) :
#   line 18 appears to contain an embedded nul
```

My session info:

```r
> sessionInfo()
R version 3.6.3 (2020-02-29)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 20.04.1 LTS

Matrix products: default
BLAS:   /usr/lib/x86_64-linux-gnu/blas/libblas.so.3.9.0
LAPACK: /usr/lib/x86_64-linux-gnu/lapack/liblapack.so.3.9.0

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8       
 [4] LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                  LC_ADDRESS=C              
[10] LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] seqinr_4.2-4        ncbi_0.0.0.9000     msa_1.18.0          Biostrings_2.54.0  
[5] XVector_0.26.0      IRanges_2.20.2      S4Vectors_0.24.4    BiocGenerics_0.32.0
[9] testthat_2.3.2     

loaded via a namespace (and not attached):
 [1] fs_1.5.0                usethis_1.6.3           devtools_2.3.2          webshot_0.5.2          
 [5] httr_1.4.2              rprojroot_1.3-2         tools_3.6.3             backports_1.1.10       
 [9] R6_2.4.1                colorspace_1.4-1        ade4_1.7-15             manipulateWidget_0.10.1
[13] withr_2.3.0             prettyunits_1.1.1       processx_3.4.4          curl_4.3               
[17] compiler_3.6.3          git2r_0.27.1            cli_2.1.0               desc_1.2.0             
[21] scales_1.1.1            readr_1.4.0             Peptides_2.4.2          callr_3.5.1            
[25] stringr_1.4.0           digest_0.6.27           rentrez_1.2.2           pkgconfig_2.0.3        
[29] htmltools_0.5.0.9001    sessioninfo_1.1.1       fastmap_1.0.1           htmlwidgets_1.5.2      
[33] rlang_0.4.8             rstudioapi_0.11         shiny_1.5.0             jsonlite_1.7.1         
[37] crosstalk_1.1.0.1       magrittr_1.5            Rcpp_1.0.5              munsell_0.5.0          
[41] fansi_0.4.1             lifecycle_0.2.0         stringi_1.5.3           whisker_0.4            
[45] MASS_7.3-51.5           zlibbioc_1.32.0         pkgbuild_1.1.0          grid_3.6.3             
[49] promises_1.1.1          crayon_1.3.4            miniUI_0.1.1.1          beautier_2.4           
[53] hms_0.5.3               knitr_1.30              ps_1.4.0                pillar_1.4.6           
[57] pkgload_1.1.0           XML_3.99-0.3            glue_1.4.2              bios2mds_1.2.3         
[61] remotes_2.2.0           vctrs_0.3.4             httpuv_1.5.4            amap_0.8-18            
[65] assertthat_0.2.1        bio3d_2.4-1             xfun_0.18               mime_0.9               
[69] xtable_1.8-4            e1071_1.7-4             later_1.1.0.1           class_7.3-15           
[73] tibble_3.0.4            memoise_1.1.0           cluster_2.1.0           rgl_0.100.54           
[77] ellipsis_0.3.1         
```

**Comment by UBod** [comment]
The problem stems from incompatibilities with newer versions of 'ape'. This has been fixed now with version 1.27.1 of the package.

**Comment by UBod** [comment]
The problem stems from incompatibilities with newer versions of 'ape'. This has been fixed now with version 1.27.1 of the package.
